configuration/cdap-site.xml

```diff
@@ -318,8 +322,8 @@
     <name>app.program.spark.yarn.client.rewrite.enabled</name>
     <value>true</value>
     <description>
-      Specify whether to rewrite the yarn.Client class in Spark to work
-      around the issue SPARK-13441 in some clusters.
+      Specify whether to rewrite the yarn.client class in Spark to work
+      around the issue SPARK-13441 in CDM clusters
```

Comment:
This was different on purpose. Why mention CDH (not CDM, defaults is wrong) on an Ambari-only use case?